src/cmdlib.py

```diff
@@ -24,8 +24,11 @@ def run_verbose(args, **kwargs):
     :raises: CalledProcessError
     """
     print("+ {}".format(subprocess.list2cmdline(args)))
-    subprocess.check_call(args, **kwargs)
-
+    try:
+        subprocess.check_call(args, **kwargs)
+    except subprocess.CalledProcessError:
+        fatal("Error running command " + args[0])
+        raise SystemExit(1)
```

Comment:
`fatal` already exits, so we shouldn't need this line.

Reply:
(Although I did learn only recently that I think it's relatively new in Python 3 that `sys.exit("Some text here")` does `fatal` which I'd been cargo-culting around since Python 2 days)